### DISCUSSION TITLE
Suggestion: Add a huge electricity pylon

### DISCUSSION THREAD

**Discussion by Sprocket10** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1524] [Q&A]
NOTE: 

Definitions of low voltage, medium voltage, etc... vary wildly from person to person, nation to nation, and standard to standard.
TAKE THE LEVELS I PROVIDE WITH A GRAIN OF SALT.

I have a suggestion to make. Limit the HE/tick of the different pylon types, increasing with size (and cost).

We already have our low voltage line (0V to 1kV) (distribution on the property):

![](https://c8.alamy.com/comp/CC72GP/overhead-low-voltage-electrical-power-line-on-wooden-pole-scout-scar-CC72GP.jpg)

![hbm lv](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/171617464/7495a328-4b53-44ad-9508-b12d329eb734)

Our medium voltage line (1kV to 36kV) (citywide distribution):

![](https://c8.alamy.com/comp/AT9MFH/medium-voltage-power-line-electricity-pylon-AT9MFH.jpg)

![hbm mv](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/171617464/863147ac-f09f-4f2c-930f-39e81d12c3d3)

And our high voltage line (36kV to 345kV) (regionwide distribution).

![](https://c1.wallpaperflare.com/preview/586/586/463/high-voltage-lines-frontline-iron-tower-cord.jpg)

![HBM HV](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/171617464/00ec63f8-539a-4b2a-97ab-06e151bac416)

I think that the mod would benefit from an extra high voltage line  (345kV to 765kV) (nationwide distribution) like so:

![](http://transglobepower.com/wp-content/uploads/2018/02/HVDC-line.jpg)

I believe that adding transfer limits would add more of a challenge and an incentive to upgrade between voltages earlier on.
Adding an ultra high voltage line (765kV and up) (international distribution) might be fun too. Here is one to look at:

![](https://cdn.chinadialogue.net/content/uploads/2020/05/20085745/W6DEX8_ultra-high_voltage_power_lines_in_China.jpg)

The people are tiny!

Thank you for reading-
Sprocket

P.S.
Million volt powerlines would go perfectly with the FEnSU!

**Comment by HbmMods** [comment]
![image](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/16674258/e023f38e-7538-445b-b4a9-42d7b0cacbff)

**Reply by djablony** [reply]
Creator, what is your opinion on adding mining durability to blocks ?

**Comment by JameH2** [comment]
Its imperative you understand that ntm as a mod, prioritizes automation. Having throughput limits on long distance easy transfer “because real life said so” is the *exact* opposite of that


If the networks was made in a way that the medium of cable drum power transfer was this arbitrary, a lot of setups would shit themselves due to the power draw/building structure they are placed in.

with all due respect, this would just cause issues.

**Reply by Sprocket10** [reply]
Thank you for the feedback. Is the only purpose of the medium and large pylons the fact that each is taller and that you need fewer to go the same distance?

**Reply by Sprocket10** [reply]
I commonly face the problem of transmission over large lakes. That is my idea for EHV pylons, even if you do not limit capacity.

**Reply by Sprocket10** [reply]
Plus, millions of volts go perfectly with stupidly large, futuristic reactors.

**Reply by Vaern** [reply]
it is against the design philosophy to tier cables. also, just put foundations for the pylon

**Comment by Sprocket10** [comment]
And as a side question, I routinely see that all suggestions are denied, however #1249 suggests an ICF reactor. And an ICF reactor was just added. Coincidence?

**Reply by Sprocket10** [reply]
Also a contributor answered #978 suggesting that the feature suggested there may be possible.

**Reply by Sprocket10** [reply]
What is this?

**Comment by JameH2** [comment]
Oh man dont argue consistency in a place where it’s usually not expected when it come to suggestions. Most suggestions are a headache and often dont respect the direction of the game.

sometimes coincedences happen, but thats like going “omg he listens to suggestions??” When i can say bob should add neptunium or something

**Reply by Sprocket10** [reply]
OK I was just curious.

**Comment by Shires53** [comment]
I see 3 possible ways to fix this issue. 
1. Add an XL/Mega Pylon that is an HBM Equivalent to an IRL VHV/UHV system. They could be 40-50 m tall and have a range of 250-1500 blocks (m). They would act the same as the "Large Electricity Pylon", except that you would actually need the DNT Transformers or even a mk-2 Substation variant for connections to other lines.
2. You could make the wire connection possibility based on block intersection, allowing long distances between current towers so long as the line doesn't touch any blocks. IRL power lines do this all the time with valleys and river crossings. 
3. combine both ideas, which is arguably the best but also most demanding option.
